lib/models/User.js

```diff
@@ -0,0 +1,33 @@
+const pool = require('../utils/pool.js');
+
+module.exports = class User {
+  id;
+  email;
+  #passwordHash;
+
+  constructor(row) {
+    this.id = row.id;
+    this.email = row.email;
+    this.#passwordHash = row.password_hash;
+  }
+
+  retrievePassword() {
```

Comment:
this works, but should technically be a "getter" function: 
```js
get passwordHash () {
return this.#passwordHash;
}
```

Reply:
the main difference from my understanding is that whereas you'll need to call `userInstance.retrievePassword()`, you will be able to dot notate your getter function: `userInstance.passwordHash` will give you the value